spec/index.html

```diff
@@ -253,44 +253,99 @@
     <section>
       <h2>Introduction</h2>
       <p>
-        JSON-LD is a JSON-based format to serialize Linked Data.
-        In recent years, YAML emerged as a more concise format
-        to represent information serialized as JSON,
+        [[JSON-LD]] is a JSON-based format to serialize Linked Data.
+        In recent years, [[YAML]] emerged as a more concise format
+        to represent information that had previously been serialized as [[JSON]],
         including API specification, data schemas, and Linked Data.
       </p>
+
+      <p>
+        This document defines YAML-LD as a set of conventions
+        on top of YAML which specify how to serialize Linked Data [[LINKED DATA]] as YAML
+        based on JSON-LD syntax, semantics, and APIs.
+      </p>
+
       <p>
         Since YAML is more expressive than JSON,
         both in the available data types and in the document structure
         (see [[I-D.ietf-httpapi-yaml-mediatypes]]),
         this document identifies constraints on YAML
-        such that it can be used to represent JSON-LD documents.
+        such that any <a>YAML-LD document</a> can be represented in JSON-LD.
       </p>
     </section>
 
   <section id="conformance">
```

Comment:
It seems like there should be some additional section breaks, at least at each H2 here (which I think should actually be H3, which will be automagically handled by Respec if the `<section>` tags are inserted appropriately), which appear to be sub-sections of `<section id="conformance">` which is IDed but not then `<hN>` titled `Conformance`....

Reply:
Our standard is to use `<section>` around every block which starts with `<Hn>`.

Reply:
I updated to use sections everywhere.